src/app/shared/actions/volume-actions/volume-actions.container.ts

```diff
@@ -39,6 +39,12 @@ export class VolumeActionsContainerComponent {
       .onErrorResumeNext()
       .filter(res => Boolean(res))
       .subscribe(() => {
+        if (!!volume.snapshots.length) {
```

Comment:
Do we really need to show 2 different dialogs here?

Reply:
We have the mock. I think it should be discussed with PM and PO

Reply:
Yes, it was discussed with PO. We need 2 dialogs here (just the same that we have for deleted VMs)